src/views/Onboarding/components/OnboardingPage.vue

```diff
@@ -77,7 +77,7 @@ export default {
   height: 100vh;
   filter: grayscale(1);
   width: 100vw;
-  background: transparent url('@/assets/images/onboarding-background.svg?inline') no-repeat scroll 0% 0%;
+  background: transparent url('@/assets/images/onboarding-background.svg?url') no-repeat scroll 0% 0%;
```

Comment:
why did this change?

Reply:
Previously, these markers were custom defined in our webpack config (rather the vue.config.js file) and named somewhat arbitrarily. Now, we’re not using webpack anymore and I just used what was provided by vite-svg-loader which now does that job. Different marker, same result.

Reply:
if the result is the same then 👍